### PR TITLE
Add circleci build

### PR DESCRIPTION
**Pull request by kimpepper**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,125 @@
+version: 2
+
+jobs:
+
+  php_56:
+    docker:
+      - image: circleci/golang:1.8
```
Comment (nickschuch) [inline .circleci/config.yml]: Whats this? :)

Comment (kimpepper) [inline .circleci/config.yml]: Not sure what image we should be building with. I think this has docker installed by default.

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+jobs:
+
+  build_image: &build_image
+    machine: true
+    steps:
+      - checkout
+      - run:
+          name: Build image
+          command: |
+            echo ${MY_SECRET}
```
Comment (nicksantamaria) [inline .circleci/config.yml]: This can be removed now.

**Comment by nicksantamaria** [comment]
LGTM. Are we likely to encounter any gotchas building images concurrently?

**Pull request merged**